geoalchemy2/functions.py

```diff
@@ -95,6 +95,35 @@ def __init__(self, *args, **kwargs):
 
 
 _FUNCTIONS = [
+
+    #
+    # Geometry Constructors
+    #
+    ('ST_MakeLine', types.Geometry,
+     'T_MakeLine comes in 3 forms: a spatial aggregate that takes rows of '
+     'point, multipoint, or line geometries and returns a line string, a '
+     'function that takes an array of point, multipoint, or line, and a '
+     'regular function that takes two point, multipoint, or line geometries. '
+     'You might want to use a subselect to order points before feeding them '
+     'to the aggregate version of this function.'
+     ''
+     'Inputs other than point, multipoint, or lines are ignored.'
+     ''
+     'When adding line components common nodes at the beginning of lines are '
+     'removed from the output. Common nodes in point and multipoint inputs are'
+     ' not removed.'
+     ''
+     'This function supports 3d and will not drop the z-index. '
+     ''
+     'Availability: 2.3.0 - Support for multipoint input elements was '
+     'introduced'
+     ''
+     'Availability: 2.0.0 - Support for linestring input elements was '
+     'introduced'
+     ''
+     'Availability: 1.4.0 - ST_MakeLine(geomarray) was introduced. ST_MakeLine'
+     ' aggregate functions was enhanced to handle more points faster.'),
+
```

Comment:
As the function docstring we just provide the short function description in the PostGIS documentation. For example for ST_MakeLine the docstring should be "Creates a Linestring from point, multipoint, or line geometries.".